onboarding/go.md

```diff
@@ -25,7 +27,7 @@ func main() {
   rollbar.SetServerHost("web.1")                       // optional override; defaults to hostname
   rollbar.SetServerRoot("github.com/heroku/myproject") // path of project (required for GitHub integration and non-project stacktrace collapsing)
 
-  result, err := DoSomething()
+  err := DoSomething()
```

Comment:
Since err is not shared at other places, you can limit the scope of the `err` within the if block.
Consider
```
err := DoSomething(); if err{
  rollbar.Critical(err)
}
```